dropwizard-validation/src/main/java/io/dropwizard/validation/PortRangeValidator.java

```diff
@@ -19,6 +19,6 @@ public void initialize(PortRange constraintAnnotation) {
 
     @Override
     public boolean isValid(Integer value, ConstraintValidatorContext context) {
-        return value == 0 || (value >= min && value <= max);
+        return value != null && (value == 0 || (value >= min && value <= max));
```

Comment:
The change causes the PortRangeValidator to _reject_ a null value. Is that intentional? It would be inconsistent with the other validators as they _accept_ a null value. I think rejecting a null value should be handled by `@NotNull`.

Reply:
@ro-kr Good catch! That was indeed not intentional. Thanks for catching this! 😄